ipatests/test_integration/test_subids.py

```diff
@@ -268,3 +269,19 @@ def test_sudid_match_shows_uid(self):
                                           f"--subuid={subuid}"])
         owner = self._parse_result(result)["owner"]
         assert owner == uid
+
+    def test_nsswitch_is_updated_with_subid_entry(self):
+        """
+        This test case checks that when ipa-client-install
+        is installed with --subid option, the nsswitch.conf
+        file is modified with the entry 'subid: sss'
+        """
+        tasks.uninstall_client(self.clients[0])
+        tasks.install_client(
+            self.master, self.clients[0], extra_args=['--subid'],
+            nameservers=None
+        )
+        cmd = self.clients[0].run_command(
+            ["grep", "subid", "/etc/nsswitch.conf"]
+        )
+        assert 'subid:      sss' in cmd.stdout_text
```

Comment:
This is probably fine but it depends on whitespace never changing and subid always being restricted to a single nss provider. I wonder if something like this would be better:
```
subid = set(cmd.stdout_text.split())
assert set(['subid:', 'sss']).issubset(subid)
```
OR if you want to ensure only one provider you can use lists instead
```
subid = cmd.stdout_text.split()
assert ['subid:', 'sss'] == subid
```

Reply:
@rcritten thank you for the review, have modified the code to use split()